debugview/src/debug/kotlin/com/revenuecat/purchases/debugview/models/InternalDebugRevenueCatScreenViewModel.kt

```diff
@@ -0,0 +1,53 @@
+package com.revenuecat.purchases.debugview.models
+
+import com.revenuecat.purchases.Purchases
+import com.revenuecat.purchases.debugview.DebugRevenueCatViewModel
+
+internal class InternalDebugRevenueCatScreenViewModel : DebugRevenueCatViewModel {
```

Comment:
I wasn't sure of whether to use the architecture components view model for this... I think we should, but it's another dependency we need to add. Probably ok though. Need to use that